packages/atomic/src/components/search/atomic-result-badge/atomic-result-badge.mdx

```diff
@@ -0,0 +1,37 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import * as AtomicResultBadgeStories from './atomic-result-badge.new.stories';
+import { AtomicDocTemplate } from '../../../../storybook-utils/documentation/atomic-doc-template';
+
+
+<Meta of={AtomicResultBadgeStories} />
+
+<AtomicDocTemplate
+  stories={AtomicResultBadgeStories}
+  githubPath="search/atomic-result-badge/atomic-result-badge.ts"
+  tagName="atomic-result-badge"
+  className="AtomicResultBadge"
+>
+
+The `atomic-result-badge` component renders a badge to highlight special features of a result.
```

Comment:
This essentially repeats the component's JSDoc (which IMO contains a bit too much information as the MDX should display the stories, which should already have examples of using the comonent with icons, slotted elements, or multi-value fields